pkg/services/ngalert/store/alert_rule.go

```diff
@@ -275,9 +275,26 @@ func (st DBstore) ListAlertRules(ctx context.Context, query *ngmodels.ListAlertR
 		q = q.Asc("namespace_uid", "rule_group", "rule_group_idx", "id")
 
 		alertRules := make([]*ngmodels.AlertRule, 0)
-		if err := q.Find(&alertRules); err != nil {
+		var faultyRules int64 = 0
```

Comment:
I think I would like us to another variable name here. I don't think `faulty` is the right word, and I was thinking we could be more precise that these are invalid definitions. In my opinion, a `faulty` rule means something else, specifically that the rule doesn't behave as intended. For example, it fires when it shouldn't.